sermo-bundle.md

```diff
@@ -3,9 +3,61 @@
 The Sermo JavaScript bundle can be included on a website and can be interfaced with `window.postMessage` (https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).
 It's a minified bundle that handles the lifecycle of the Sermo chat window.
 
-## How to start a sales chat
+## Setup
 
-Send a message through `window.postMessage` with a payload of this format:
+Include a script that with any of these scripts within the head block of your site.
+
+| env  | host |
+| -    | -    |
+| dev  | https://sermo-webchat-ci1.sermo.dev-dockeree.int.comhem.com/bundle/sermo.min.js |
+| prod | https://sermo.comhem.com/bundle/sermo.min.js |
+
+All interactions with the sermo bundle is done through window.postMessage. The sender of each message should be the brand of the site that is sending the message.
+
+Available brands are
+
+- comhem
+- comviq
+- boxer
+
+## Starting chats
+
+To start a chat a message is sent through `window.postMessage` with a payload that corresponds to the type of chat you want to start.
+
+### Common
+
+The common parts for all events are:
+
+```json
+{
+  "sender": "WEBSITE_BRAND",
+  "receiver": "sermo",
+  "event": "...",
+  "customerId": "12345678",
+  "maxZIndex": 1501
+  ...
+}
+```
+
+Where the `WEBSITE_BRAND` string should be the brand of the website. 
+
+The value of `customerId` should be provided if the customer is logged in on the site. Doing so enables the bot to not ask for it and only request the intent of the customer. This is an optional field.
+
+The value for `maxZIndex` is used so that we can make sure that the CSS `z-index` we open the iFrame with is the highest on the page.
```

Comment:
![image](https://user-images.githubusercontent.com/5889538/91028474-ce5e6200-e5fd-11ea-82e9-b265d63015c3.png)


Maybe it's good to mention that this is optional as well?